static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -136,19 +182,23 @@
     }
 }
 
-.new-header__burger-icon--open {
-    span {
-        &:nth-child(1) {
-            opacity: 0;
-        }
-        &:nth-child(2) {
-            transform: rotate(45deg);
-        }
-        &:nth-child(3) {
-            transform: rotate(-45deg);
-        }
-        &:nth-child(4) {
-            opacity: 0;
+.new-header__nav__menu-button--open {
+    transform: scale(1.1);
+
+    .new-header__burger-icon {
+        span {
```

Comment:
Should try and not style on elements. Also, no need to nest - you're able to do `.new-header__burger-icon .new-class` if you need the specificity (bearing in mind this is three levels deep now).